capture-sdk/sdk/src/doc/source/migrate-to-3-0-0.rst

```diff
@@ -232,6 +232,15 @@ Breaking Changes
 
 QR code scanning UI and functionality are changed. Scanning and processing happens automatically. You can find more details `here <features.html#qr-code-scanning>`_.
 
+Review
+~~~~~
+
+Breaking Changes
+++++++++++++++++
+
+Review screen doesn't support re-ordering the images anymore.
```

Comment:
[Request] Image rotation is also not supported anymore. The Gini Pay API can automatically correct rotation during processing.